test/dispatcher.js

```diff
@@ -1,7 +1,19 @@
+define([ 'intern!tdd',
+        'intern/chai!',
```

Comment:
That's exactly what's listed here. But you're parameters in the callback don't match the list of dependencies.